### ISSUE TITLE
Errors when installing raymarching

### ISSUE THREAD

**Issue by zhuole1025** [ashawkey/stable-dreamfusion#214]
### Description

After updating the code, i get the following error with `raymarching` every time I run the `main.py`. So, I have to rerun `pip install ./raymarching` to solve this problem. Is there something run with my installation or enviroment?
```
TypeError: composite_rays_train_forward(): incompatible function arguments. The following argument types are supported:
    1. (arg0: at::Tensor, arg1: at::Tensor, arg2: at::Tensor, arg3: at::Tensor, arg4: int, arg5: int, arg6: float, arg7: bool, arg8: 
at::Tensor, arg9: at::Tensor, arg10: at::Tensor, arg11: at::Tensor) -> None
```

### Steps to Reproduce

```
python main.py xxxxxx
```

### Expected Behavior

run without the above error

### Environment

Ubuntu 22.04, pytorch 1.10.2

**Comment by ashawkey** [comment]
@zhuole1025 Hi, do you mean you have to reinstall raymarching everytime you run a command?

**Comment by zhuole1025** [comment]
@ashawkey right. I open two shells to run the training script, and every time I run one a command in one shell, then I have to reinstall raymarching in another shell. This is weird..

**Comment by ashawkey** [comment]
Could you make sure the code under two shells are both the latest?

**Comment by zhuole1025** [comment]
@ashawkey Yes, they are in the same folder..
